internal/querycoord/meta.go

```diff
@@ -618,26 +603,38 @@ func (m *MetaReplica) saveGlobalSealedSegInfos(saves col2SegmentInfos) (col2Seal
 					}
```

Comment:
no need to set onlineNodeIDs? If there is no online segments/ online nodeID, how does querynode automatically clean up growing segments?